src/components/Footer.tsx

```diff
@@ -0,0 +1,54 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  filter: string,
+  todos: Todo[]
+  setFilter: (text: string) => void;
+}
+
+export const Footer: React.FC<Props> = ({ filter, todos, setFilter }) => {
```

Comment:
Good enough!

Take a look that, according to [Demo Page](https://mate-academy.github.io/react_todo-app-with-api/), the quantity of the items left should stay unchanged regardless of the filter we use.


![230703-review-1](https://github.com/mate-academy/react_todo-app-loading-todos/assets/111086548/102f616e-b503-4447-a13b-f7e700030de5)
![230703-review-2](https://github.com/mate-academy/react_todo-app-loading-todos/assets/111086548/73b7692d-1354-402d-8e7f-e2bc01db2f7b)
![230703-review-3](https://github.com/mate-academy/react_todo-app-loading-todos/assets/111086548/f5d252b2-e479-43b6-99fc-a5b0ee5d0bc9)